cfe_internal/CFE_cfengine.cf

```diff
@@ -31,6 +31,14 @@ bundle agent cfe_internal_management
 
     am_policy_hub.enterprise::
 
+      "hub" -> { "Mission Portal", "CFEngine Enterprise" }
+        usebundle => cfe_internal_purge_scheduled_reports_older_than_days( $(def.purge_scheduled_reports_older_than_days) ),
+        ifvarclass => "enable_cfe_internal_purge_scheduled_reports_older_than_days",
```

Comment:
Granted, this is probably not a good example since you probably wouldnt have 10k hubs. But its a general pattern that I was trying to establish. Things ought to be specifically enabled.